src/store/asana/index.ts

```diff
@@ -147,6 +152,10 @@ export const useAsanaStore = defineStore("asana", {
 
     MERGE_TASKS(payload: Task[]): void {
       // replace individual task with each task in payload
+      if (this.reloadLock.locked ||
```

Comment:
The second part of this if statement should probably in a variable with a descriptive name. That way its intent becomes clear to others